content/plugins/using-plugins.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: bt_wiki
+title: Working With Plugins
+category: Docs
+draft: false
+weight: 50
+
+---
+
+Cloudify Manager exposes a plugins API, used to allow the installation of Python dependencies from pre-packaged Python wheels. The plugins could come in handy for the following:
+
+* Allowing installation of Python dependencies without needing access to PyPi or the internet.
+* Eliminating the need for system level compilation tools such as gcc.
+
+### Creating A Plugin Package
+
+The official Cloudify tool used to create plugin packages is named [Wagon](https://github.com/cloudify-cosmo/wagon). In addition to the required package wheels, plugin packages created by Wagon also contain metadata regarding the plugin such as package name, compiled distribution, plugin package version and more.
```

Comment:
compiled distribution -> the distribution the plugin was compiled on (if not Pure Python)